plugins/newsdownloader.koplugin/main.lua

```diff
@@ -105,6 +106,15 @@ function NewsDownloader:addToMainMenu(menu_items)
                 keep_menu_open = true,
                 callback = function() self:removeNewsButKeepFeedConfig() end,
             },
+            {
+                text = _("Global skip images download"),
```

Comment:
If you reverse the logic you could just call it:

```suggestion
                text = _("Download images"),
```

Or maybe:
```suggestion
                text = _("Download images (default)"),
```